1.3/spec.md

```diff
@@ -0,0 +1,222 @@
+# MBTiles 1.3
+
+## Abstract
+
+MBTiles is a specification for storing tiled map data in
+[SQLite](http://sqlite.org/) databases for immediate usage and for transfer.
+MBTiles files, known as **tilesets**, must implement the specification below
+to ensure compatibility with devices.
+
+## Database Specifications
+
+Tilesets are expected to be valid SQLite databases of
+[version 3.0.0](http://sqlite.org/formatchng.html) or higher.
+Only core SQLite features are permitted; tilesets **cannot require extensions**.
+
+MBTiles databases can optionally use [the officially assigned magic number](http://www.sqlite.org/src/artifact?ci=trunk&filename=magic.txt)
+to be easily identified as MBTiles.
+
+## Database
+
+Note: the schemas outlined are meant to be followed as interfaces.
+SQLite views that produce compatible results are equally valid.
+For convenience, this specification refers to tables and virtual
+tables (views) as tables.
+
+### Metadata
+
+#### Schema
+
+The database is required to contain a table or view named `metadata`.
+
+This table must yield exactly two columns named `name` and
+`value`. A typical create statement for the `metadata` table:
+
+    CREATE TABLE metadata (name text, value text);
+
+It is common to create an index for this table:
+
+    CREATE UNIQUE INDEX name on metadata (name);
+
+#### Content
+
+The metadata table is used as a key/value store for settings. Two keys are **required**:
+
+* `name`: The plain-English name of the tileset.
+* `format`: The file format of the tile data.
+
+For new tilesets, the `format` should be a MIME type: for example,
+`image/png` or `image/jpeg` for PNG or JPEG image data, or
+`application/vnd.mapbox-vector-tile` for
+[Mapbox Vector Tile](https://github.com/mapbox/vector-tile-spec/) data.
+
+Previous versions of the MBTiles spec called for PNG and JPEG data
+to be tagged with a `format` of `png` or `jpg`, so these may be found
+in existing tilesets and should be treated as synonyms for
+`image/png` and `image/jpeg`. Many existing Mapbox Vector Tile tilesets
```

Comment:
I included this because [mbutil](https://github.com/mapbox/mbutil/blob/master/mbutil/util.py#L153) inserts a metadata row for format=png. My only goal with this PR is to document what existing implementations do.